sklearn/ensemble/tests/test_voting.py

```diff
@@ -109,11 +133,26 @@ def test_tie_situation():
     assert_equal(eclf.fit(X, y).predict(X)[73], 1)
 
 
+def test_average_prediction():
+    """ Check AveragingRegressor with multiple models"""
+    reg1 = BaggingRegressor(base_estimator=DecisionTreeRegressor(),
+                            random_state=1,  n_estimators=100)
+    reg2 = RandomForestRegressor(random_state=1, n_estimators=100)
+    reg3 = GradientBoostingRegressor(random_state=1, n_estimators=100)
+
+    ereg = AveragingRegressor([('br', reg1), ('rf', reg2), ('gbr', reg3)])
+
+    scores = cross_val_score(ereg, X_r, y_r, cv=5, scoring='r2')
+    assert_almost_equal(scores.mean(), 0.65, decimal=2)
```

Comment:
I don't know why this should be the test value. Did you calculate it by hand? If any one of the constituent estimators changes, this result and all similar ones will have to change. Rather you should test not against  some hard coded score but by checking against some simple logic for this specific set of estimators.

I also think using cross_val_score is unnecessarily complex for a test of basic functionality... And if the test failed that complexity makes it harder to debug.

Reply:
Even using DummyRegressor would give a clearer test of basic functionality

Reply:
changed to DummyRegressor